### PR TITLE
Adds Corporate Account Executive job ad

### PR DESCRIPTION
**Pull request by gabrielamatias**

### CODE REVIEW

**Comment by gabrielamatias** [comment]
@jesssummerfield , @jacobpgn : please review.

File: app/pages/about/jobs/positions/corporate-account-executive.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/corporate-account-executive.js]: :scissors: the blank line

File: app/pages/about/jobs/positions/corporate-account-executive.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          model is aligned with the success of our customers: their success in getting paid links
+          to our rewards.
+        </p>
+        <p className='para'>
+          In order to navigate a path to mutual success you will need to develop a good
+          understanding of our solution, the business value, and technical fit and its
+          relationship to our customers’ existing systems.
+        </p>
+        <p className='para'>
+          We've built a dynamic, collaborative culture in which we hire and develop smart,
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/corporate-account-executive.js]: `'` -> '’'

**Comment by jacobpgn** [comment]
:+1: just some tiny comments :tropical_fish:

File: app/pages/about/jobs/positions/corporate-account-executive.js (hunk 2 above)
Comment (jacobpgn) [inline app/pages/about/jobs/positions/corporate-account-executive.js]: Looks like they're all `'` now, that'll do!

**Comment by gabrielamatias** [comment]
Closing as I've put a new PR.

**Pull request closed without merging**